src/freertos_drivers/esp32/Esp32HardwareTwai.cxx

```diff
@@ -37,12 +37,9 @@
 
 // Ensure we only compile this code for the ESP32 family of MCUs and that the
 // ESP-IDF version is supported for this code.
```

Comment:
we can truncate this comment after MCUs.

Reply:
done